mods/common/metrics.yaml

```diff
@@ -46,3 +46,8 @@ Metrics:
 	NoticeWarningColor: FFA500
 	NoticeSuccessColor: 00FF00
 	NoticeErrorColor: FF0000
+	WidgetDefaultCursor: default
```

Comment:
Widget and Ingame are broadly kind of okay, but IMO don't properly describe the way we actually use these which might cause trouble in the future if we want to add more types.

How about:
```
DefaultCursor: default
ButtonCursor: default
WorldDefaultCursor: default
WorldSelectCursor: default
```
?

Reply:
I like your naming scheme better. Hope I got it right.